### ISSUE TITLE
Unit test fail 'ACL HTTP Origin Should PUT new ACL file'

### ISSUE THREAD

**Issue by dmitrizagidulin** [linkeddata/ldnode#172]
Cloned repo, did `npm install`. Then ran the tests, `npm test`. One test is failing:

```
  1) ACL HTTP Origin Should PUT new ACL file:
     Uncaught AssertionError: expected 403 to equal 201
      at Request._callback (test/acl.js:243:24)
      at Request.self.callback (node_modules/request/request.js:198:22)
      at Request.<anonymous> (node_modules/request/request.js:1035:10)
      at IncomingMessage.<anonymous> (node_modules/request/request.js:962:12)
      at _stream_readable.js:908:16
```

Am I missing any setup or configuration steps?

(Let me know if you need any debug log messages).

**Comment by nicola** [comment]
Ha! After a few inspection we basically create some files that we never remove, so if you run the test twice the file is already there in place, hence this test fails because you can't create something that is already there.

So, an easy fix would be to remove the file/folder at the beginning of the test

**Comment by dmitrizagidulin** [comment]
@nicola - I notice that the `test/acl.js` Cleanup step was changed to skipped, here: https://github.com/linkeddata/ldnode/commit/cd42e336e397af3b535a0ad77d5cddef680f8f1b#diff-c65e0d598fa4659ff66a65afb2057ee3L1044

Was that intentional? The `acl` tests don't seem to clean up after themselves now (here's my `git status` after the tests run):

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	test/resources/acl/origin/test-folder/
	test/resources/acl/write-acl/default-for-new/
	test/resources/acl/write-acl/empty-acl/another-empty-folder/
	test/resources/acl/write-acl/empty-acl/test-file
	test/resources/acl/write-acl/empty-acl/test-folder/
	test/resources/acl/write-acl/test-file
	test/resources/acl/write-acl/test-file.acl
```

Do we need to re-do the cleanup code, or add the `test/resources/` dir to `.gitignore`?

**Comment by nicola** [comment]
Thanks for pointing this out. I plan to fix this issue, once done with the #164.
The cleanup step has been commented because it is from the old tests (soon to be deleted) - that I refactored - mostly completely - just before christmas.

I would not put them on `.gitignore`, I would just make sure I remove the folders and files once the test is done (or if possible at each test, in order to make the tests independent of each other) and put a cleanup function at the beginning of the test (not as a test, but as a `before()`)

**Comment by nicola** [comment]
This is solved by https://github.com/linkeddata/ldnode/commit/b76a9f2e55b54beb4c7b8ab5a7d2821335aa86f3
